public/app/features/apiserver/server.ts

```diff
@@ -69,7 +75,7 @@ export class ScopedResourceServer<T = object, K = string> implements ResourceSer
 
           case 'in':
           case 'notin':
-            return `${key}${operator}(${label.value.join(',')})`;
+            return `${key} ${operator} (${label.value.join(',')})`;
```

Comment:
Is this change required? I see a similar line for = / != operators above that was not changed.